component-classes/index.js

```diff
@@ -414,6 +414,7 @@ export const clickable = {
   clickable: 'absolute inset-0 h-full w-full appearance-none cursor-pointer focusable focusable-inset',
   clickableNotToggle: 'inset-0 absolute',
   label: `px-12 ${label.label} py-8! cursor-pointer focusable focusable-inset`,
+  focusable: 'focusable',
```

Comment:
this is not needed as here it is already being set

Reply:
Check for similar fix but in vue  https://github.com/warp-ds/vue/pull/38/files

Reply:
My bad, I should've elaborated that I needed just `focusable` class on the button/a elements of Clickable. It was because `focusable-inset`, which is set on other clickable classes, was making the outline stick too much to the text. I still might be missing something here - comparing this component with other packages and Fabric Clickable component was not as easy as I had anticipated 😰